WorkflowUI/Sources/Screen/AdaptedEnvironmentScreen.swift

```diff
@@ -30,7 +30,7 @@ import ViewEnvironment
 ///     .adaptedEnvironment(keyPath: \.myValue, to: newValue)
 /// ```
 ///
-public struct AdaptedEnvironmentScreen<Content: Screen>: Screen {
+public struct AdaptedEnvironmentScreen<Content> {
     /// The screen wrapped by this screen.
     public var wrapped: Content
```

Comment:
nit: the parameter names & comments now seem slightly mis-matched if the generic type is unconstrained

Reply:
I think in reality, this is going to be 99% screens / mapped to one eventually, so I left it as is.